utils/index.js

```diff
@@ -187,6 +187,18 @@ module.exports.ckanViewToDataPackageView = (ckanView) => {
   return dataPackageView
 }
 
+module.exports.ckanSchemaToTableSchema = (ckanSchema) => {
+  let fields = {}
+  fields.name = ckanSchema.id
+  fields.type = ckanSchema.type
```

Comment:
We need to convert datastore type into tableschema type. Here is the dictionary you can re-use:

```
const FIELDS = [
  {
    datastore: 'text',
    tableschema: 'string'
  },
  {
    datastore: 'int',
    tableschema: 'integer'
  },
  {
    datastore: 'float',
    tableschema: 'number'
  },
  {
    datastore: 'date',
    tableschema: 'date'
  },
  {
    datastore: 'time',
    tableschema: 'time'
  },
  {
    datastore: 'timestamp',
    tableschema: 'datetime'
  },
  {
    datastore: 'timestamp',
    tableschema: 'datetime'
  },
  {
    datastore: 'bool',
    tableschema: 'boolean'
  },
  {
    datastore: 'json',
    tableschema: 'object'
  }
]
```

Reply:
so, we don't need to add a `type` input field on the backend template. ??

Reply:
@sagargg you still need it. This suggestion is for making this work with other projects.

Reply:
@anuveyatsu but the `type` is overridden form from info property of datastore.  which is `string ` by default.